content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -328,14 +328,24 @@ before liveness or readiness probes are initiated. Defaults to 0 seconds. Minimu
 * `periodSeconds`: How often (in seconds) to perform the probe. Default to 10
 seconds. Minimum value is 1.
 * `timeoutSeconds`: Number of seconds after which the probe times out. Defaults
-to 1 second. Minimum value is 1.
+to 1 second. Minimum value is 1. For exec probes, the process inside the container
+may keep running even after probe returned failure because of the timeout. Incorrect
+implementation of readiness probe may result in ever growing number of processes
+in the container and resource starvation.
 * `successThreshold`: Minimum consecutive successes for the probe to be
 considered successful after having failed. Defaults to 1. Must be 1 for
 liveness. Minimum value is 1.
 * `failureThreshold`: When a probe fails, Kubernetes will
 try `failureThreshold` times before giving up. Giving up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready.
 Defaults to 3. Minimum value is 1.
 
+{{< note >}}
+Before the version `1.20` the field `timeoutSeconds` was not respected for exec probes and probe never returned error when deadline was exceeded. So these probes may run indefinitely.
```

Comment:
The probe timeout is now respected, but in dockershim we don't stop the probe from running indefinitely, not sure we need to clarify that.

Reply:
added clarificaton and shortened the sentence